### PR TITLE
Make issue template more enthusiastic

### PR DESCRIPTION
**Pull request by RyanCavanaugh**
🤷‍♂️

### CODE REVIEW

**Comment by mhegazy** [comment]
:D

File: issue_template.md [hunk 1]
```diff
@@ -1,10 +1,33 @@
-<!-- BUGS: Please use this template. -->
-<!-- QUESTIONS: This is not a general support forum! Ask Qs at http://stackoverflow.com/questions/tagged/typescript -->
-<!-- SUGGESTIONS: See https://github.com/Microsoft/TypeScript-wiki/blob/master/Writing-Good-Design-Proposals.md -->
+<!-- 🚨 𝑺𝑻𝑶𝑷 🚨 𝑺𝑻𝑶𝑷 🚨 𝑺𝑻𝑶𝑷 🚨 -->
```
Comment (DanielRosenwasser) [inline issue_template.md]: As much as they should be, your italicized Unicode letters may not be visible for all  You should have a fallback for these. Maybe alternate 𝑺𝑻𝑶𝑷 and STOP for good measure?

File: issue_template.md [hunk 2]
```diff
@@ -1,10 +1,33 @@
-<!-- BUGS: Please use this template. -->
-<!-- QUESTIONS: This is not a general support forum! Ask Qs at http://stackoverflow.com/questions/tagged/typescript -->
-<!-- SUGGESTIONS: See https://github.com/Microsoft/TypeScript-wiki/blob/master/Writing-Good-Design-Proposals.md -->
+<!-- 🚨 𝑺𝑻𝑶𝑷 🚨 𝑺𝑻𝑶𝑷 🚨 𝑺𝑻𝑶𝑷 🚨 -->
+<!--
+Half of all issues filed here are duplicates, answered in the FAQ, or not appropriate for the bug tracker.
+
+Please help us by doing the following steps before logging an issue:
```
Comment (DanielRosenwasser) [inline issue_template.md]: by doing -> by taking

File: issue_template.md [hunk 3]
```diff
@@ -1,10 +1,33 @@
...
+  * Search: https://github.com/Microsoft/TypeScript/search?type=Issues
+  * Read the CONTRIBUTING guidelines: https://github.com/Microsoft/TypeScript/blob/master/CONTRIBUTING.md
+  * Read the FAQ: https://github.com/Microsoft/TypeScript/wiki/FAQ
+-->
+
+<!-- If you have a QUESTION:
+   THIS IS NOT A FORUM FOR QUESTIONS.
+   Ask questions at http://stackoverflow.com/questions/tagged/typescript
+-->
```
Comment (DanielRosenwasser) [inline issue_template.md]: Point people to Gitter too?

**Comment by RyanCavanaugh** [comment]
Surprisingly, this worked? The noise:bug ratio has gone from 3:2 to 1:1. The SNR hasn't been this good since 2014. Unactionable reports are also down ~30%.

/cc @jcalz

**Pull request merged**